ui/app/LICENSE

```diff
@@ -0,0 +1,29 @@
+BSD 3-Clause License
```

Comment:
I'm certainly no legal expert, but my guess would be that from a legal perspective the relicensing/ownership needs to be done for this PR already.